docs/source/using/managing-minishift.adoc

```diff
@@ -104,7 +73,7 @@ places all runtime state into `~/.minishift`. This environment variable is
 currently experimental and semantics might change in future releases.
 
 [[persistent-configuration]]
-=== Persistent configuration
+== Persistent configuration
```

Comment:
I was trying to avoid deeper nesting and this way the persistent options and volumes show up in the topic TOC. If you don't think this is useful I can move them back down one level.